File: 251001/Lashkin/publisher/DTO/ResponseDTO/NewsResponseDTO.cs
﻿using Publisher.Models;

namespace Publisher.DTO.ResponseDTO;

public class NewsResponseDTO
{
    public long Id { get; set; }
    public string Title { get; set; }
    
    public long UserId { get; set; }
    public User User { get; set; }

    public string Content { get; set; }
    public DateTime Created { get; set; }
    public DateTime Modified { get; set; }

    public List<Label> Labels { get; set; } = [];
}

File: 251003/Dedov/Lab3/publisher/src/main/java/by/bsuir/publisher/dto/responses/NewsResponseDto.java
package by.bsuir.publisher.dto.responses;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

import java.time.LocalDateTime;

@SuperBuilder
@NoArgsConstructor
@Data
@EqualsAndHashCode(callSuper = true)
public class NewsResponseDto extends BaseResponseDto {
    private Long editorId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
}
